psychiatric_guide_app/settings.py

```diff
@@ -143,8 +143,12 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = 'static'
+STATIC_ROOT = ''
```

Comment:
this needs to be set as 'static' for app engine

Reply:
Added: 'static' to STATIC_ROOT